src/Jackett/packages.config

```diff
@@ -1,7 +1,7 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="CsQuery" version="1.3.4" targetFramework="net451" />
-  <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net451" />
-  <package id="NLog" version="4.0.1" targetFramework="net451" />
-  <package id="NLog.Windows.Forms" version="2.0.0.0" targetFramework="net451" />
-</packages>
\ No newline at end of file
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Same with this file, looks like nothing changed though so can you remove it from the pull?